pyproject.toml

```diff
@@ -73,6 +73,7 @@ filterwarnings = [
   'ignore:\n.*Sentinel is not a public part of the traitlets API.*:DeprecationWarning',
   "ignore:.*metadata to be logged in the data array, it is necessary to install the orsopy package.:UserWarning",
   "ignore:.*A transformation needs a vector attribute. Falling back to returning underlying value.:UserWarning",
+  "ignore:.*Invalid transformation, missing attribute.*:UserWarning",
```

Comment:
Would it be possible to do this locally in the tests? I presume it affects only Amor files?

Reply:
Yes you're right.
Maybe this will already be resolved by https://github.com/scipp/essreflectometry/pull/89/commits/eda8d9b4637f4e4406f483af780b78bb778fcea0 when that is merged? What do you think?

Reply:
Not sure, does this mean we ignore all those warnings whenever we somehow import that module?

Reply:
That's the intention. This is the thought process: we only import the module when loading amor files, the amor files are broken in a known way and we don't want the warning to show when we are loading them.

Reply:
> That's the intention at least

My point is that we should *not* do that. If we run something that for some reason also imports `ess.amor`, we shouldn't ignore errors everywhere else (say when we run `estia`), right?

Reply:
That makes sense, I've updated the referenced PR to be more selective about what warnings are ignored